tests/layers/test_numerical_calibrator.py

```diff
@@ -839,3 +894,51 @@ def test_training():
     assert torch.allclose(
         torch.absolute(keypoints_inputs), keypoints_outputs, atol=2e-2
     )
+
+
+def test_training_learned_interior():
+    """Tests that the `NumericalCalibrator` can learn interior keypoints on a PWL."""
```

Comment:
I would update this to mention the specific function that it's trying to learn. If you need to utilize additional docstring lines below the top level one due to space issues, feel free to do so. I think it's worth clarifying here in more detail how you're testing this (i.e. the function you're learning and why testing for that function tests that we're learning properly).